Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -192,6 +192,10 @@ def prepare(self,mReader, directories, mode='init'):
             wmsplit['RECOUP15']=5
             wmsplit['RECOAODUP15']=5
             wmsplit['DBLMINIAODMCUP15NODQM']=5
+            wmsplit['DigiFull']=5
+            wmsplit['RecoFull']=5
+            wmsplit['DigiFullPU']=1
+            wmsplit['RecoFullPU']=1
```

Comment:
first two lines are for noPU workflows 5 lumis/job, last two lines are for PU workflows 1 lumi/job. I just take the same configs as for 2015 workflows.